docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2806,6 +2806,13 @@ If a simulant has more than 4 dependents,
 4 of their dependents are chosen to be included on the 1040 and the rest are omitted.
 This can be uniformly at random (preferred), or in another way if that is easier computationally.
 
+.. note::
+  Due to random sampling of a filer's dependents being more complicated to implement, the engineers have
+  currently implemented this as the 4 dependents included in the 1040 observer as the first 4 of the filer,
+  rather than 4 randomly selected dependents from both guardians in a joint filing row. In a later release,
```

Comment:
I thought it would be a little more complicated than this --- first four of the filer, if the filer has four or more dependents.  Otherwise, all dependents of filer, plus additional dependents of joint filer to a maximum of four total.

Reply:
I think the idea here was that there wouldn't be much distinction for joint filers of whose dependent was whose. We wanted to do random sampling to maximize the chance of getting different dependents on 1040s from different years, which creates the most ER challenge.